firebase.go

```diff
@@ -173,6 +173,16 @@ func (fb *Firebase) doRequest(method string, body []byte) ([]byte, error) {
 	}
 
 	resp, err := fb.client.Do(req)
+	if err == nil && resp.StatusCode == http.StatusTemporaryRedirect {
```

Comment:
yeah. missed the case nil hidden in there. i figured nil wasn't checked and it fell through.